src/Windows/Avalonia.Win32/WinRT/WinRTApiInformation.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System;
+using Avalonia.Logging;
+using MicroCom.Runtime;
+
+namespace Avalonia.Win32.WinRT;
+
+/// <summary>
+/// Any WinRT API might not be available even if Windows version is supposed to support them (Win PE, Xbox...).
+/// Using ApiInformation is a typical solution in UWP/WinUI apps, so we should do as well.
+/// </summary>
+internal static unsafe class WinRTApiInformation
```

Comment:
This API could also be used in our WinRT Compositor classes to check where it is supported.  But compositor APIs are a bit more complicated, as they are available pretty early for UWP, but Win32 handles support was introduced pretty late.